tests/test_dpfresultwithkeywords.py

```diff
@@ -233,12 +235,37 @@ def test_groupingelshape_nodallocation(allkindofcomplexity):
     disp = d.vector
     assert disp.num_fields == 4
     assert disp.result_fields_container.get_label_space(3) == {'elshape': 3, 'time': 1}
-    assert len(disp.get_data_at_field(0)) == 93950
-    assert len(disp.get_data_at_field(1)) == 6748
-    assert len(disp.get_data_at_field(2)) == 36
+    assert len(disp.get_data_at_field(0)) == 14826
+    assert len(disp.get_data_at_field(1)) == 1486
+    assert len(disp.get_data_at_field(2)) == 19
     assert len(disp.get_data_at_field(3)) == 4
     assert np.isclose(disp.get_data_at_field(2)[0][0], 5.523488975819807e-20)
     assert disp[0].location == locations.nodal
+    
+    disp.plot_contour('elshape', 1)
+    
+    # with dpf.core operator
```

Comment:
Consider breaking this test up here since it's quite long and each test should test an individual API.